lib/generators/react_on_rails/generator_helper.rb

```diff
@@ -1,8 +1,13 @@
 # frozen_string_literal: true
 
+require "package_json"
 require "rainbow"
 
 module GeneratorHelper
+  def package_json
+    @package_json ||= PackageJson.read(Rails.root.to_s)
```

Comment:
@G-Rath I've changed it vs how it was implemented [here](https://github.com/G-Rath/react_on_rails/commit/6c0a29ebff3b584b6446f1065dec23b6bf7ffb50#diff-84d1b667e35dc8d3cf92e40813c3b880c84a1d31dbd6d6367e8851f84418b435R6) cause it was creating a separate bundle context, due to which I couldn't locate the shakapacker gem during the install generator work.